pai-management/paictl.py

```diff
@@ -20,6 +20,8 @@
 from __future__ import print_function
 
 import sys
+reload(sys)
+sys.setdefaultencoding('utf-8')
```

Comment:
Maybe instead, we should fix this error by using [codecs](https://docs.python.org/2/library/codecs.html#codecs.open) to open template file instead of plain open.

Rational:

If we setdefaultencoding here, maybe other python scripts should also add these two lines to prevent potential unicode error.

Reply:
Agree with xudifsd. The 'setdefaultencoding' method has several other flaws, such as overriding python environment encoding setting, encouraging developers to mis-use str, etc. See https://stackoverflow.com/questions/3828723/why-should-we-not-use-sys-setdefaultencodingutf-8-in-a-py-script.